app/controllers/albums.js

```diff
@@ -0,0 +1,18 @@
+const albumsService = require('../services/albums');
+
+const getAlbums = (req, res, next) => {
+  albumsService
+    .getAlbums()
+    .then(albums => res.json(albums))
```

Comment:
i suggest you use `res.status(200).send(response)` intead of `res.json()`

you can read this for more information https://blog.fullstacktraining.com/res-json-vs-res-send-vs-res-end-in-express/